frontend/app/components/api/api-v3/hal-resource-dms/payload-dm.service.ts

```diff
@@ -58,7 +58,7 @@ export class PayloadDmService {
     }
 
     _.each(nonLinkProperties, property => {
-      if (resource[property]) {
+      if (resource.hasOwnProperty(property)) {
```

Comment:
/cc @ulferts This caused saving a query with any `false` value (e.g., `displaySums: false` or, in this case, timelineVisible) to ignore that value.